build/create_release_notes.py

```diff
@@ -0,0 +1,219 @@
+    info = json.loads(raw_info)
+    if (pr_cache is not None):
+        with open(pr_cache + "/" + commit_hash + ".json", 'w') as fout:
+            json.dump(info, fout)
+    return info
+
+def get_prs(commits, pr_cache):
+    return [(get_pr(commit[0], pr_cache), commit) for commit in commits]
+
+def dedup_prs(prs):
+    found = set()
+    dedupped = []
+    for pr, commit in prs:
+        if pr is None or len(pr) == 0:
+            print("No PR: " + str(commit))
+        else:
+            if len(pr) != 1:
+                print("Too many PRs " + str(pr) + " " + str(commit))
+            else:
+                if not pr[0]['url'] in found:
+                    found.add(pr[0]['url'])
+                    dedupped.append((pr, commit))
+    return dedupped
+
+def get_category(pr, label_config, commit):
+    main_label = None
+    label_names = [label['name'] for label in pr['labels']]
+    for category in label_config['categories']:
+        for label in category['labels']:
+            if label in label_names:
+                return category['title']
+    print(f"No label: {pr['html_url']} {str(label_names)} {str(commit)}")
+    return label_config['catch_all']
+
+def generate_note(prs, commit, label_config):
+    if len(prs) == 0:
+        return ("Direct Commit", commit[1])
+    if len(prs) > 1:
+        print("Too many PRs?")
+    pr = prs[0]
+    category = get_category(pr, label_config, commit)
+    text = '* ' + pr['title'] + " by @" + pr['user']['login'] + ' in ' + pr['html_url']
+    return (category, text)
+
+def format_notes(notes, label_config, old_version, new_version):
+    grouping = defaultdict(list)
+    for (category, line) in notes:
+        grouping[category].append(line)
+    text = ''
+    categories = label_config['categories'] + [{"title": label_config['catch_all']}]
+    for category in categories:
+        title = category['title']
+        if title in grouping:
+            notes = grouping[title]
+            put_in_summary = category.get('collapsed', False)
+            header = f"#### {title}"
+            if put_in_summary:
+                text += f"\n<details>\n<summary>\n\n{header} (click to expand)\n\n</summary>\n\n"
+            else:
+                text += f'{header}\n'
+            for note in notes:
+                text += f"{note}\n"
+            if put_in_summary:
+                text += '\n</details>\n'
+    text += f"\n\n**Full Changelog**: https://github.com/FoundationDB/fdb-record-layer/compare/{old_version}...{new_version}"
+    text += f"\n\n<!-- MIXED_MODE_RESULTS {new_version} PLACEHOLDER -->\n"
+    return text
+
+def replace_note(lines, note):
+    print(f"Inserting note {note.old_version} -> {note.new_version}")
+    new_lines = []
+    added = False
+    for line in lines:
+        if not added and line == note.old_version_header():
+            new_lines.append(note.new_version_header())
```

Comment:
```suggestion
            new_lines.append(note.new_version_header())
            new_lines.append('')
```

So that we get a new line between new version header and the release notes